tests/molecular/topology_graphs/vertex/placement/cases/metal_complex/fixtures/monodentate.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import numpy as np
+import stk
+from functools import partial
+from scipy.spatial.distance import euclidean
+
+from ....case_data import CaseData
+
+vertices = stk.metal_complex.vertices
+
+
+@pytest.fixture
+def monodentate(position, building_block_1):
+
+    point1 = points = (position + [-10, 0, 0], )
+
+    def get_fg0_point(building_block):
+        return get_closest_point(
+            points=points,
+            point=get_fg_position(0, building_block),
+        )
+
+    vertex = vertices._MonoDentateLigandVertex(
+        id=0,
+        position=position,
+    )
+
+    return CaseData(
+        vertex=vertex,
+        edges=tuple(get_edge(vertex)),
+        building_block=building_block_1,
+        position=position,
+        alignment_tests={get_fg0_point: point1},
```

Comment:
There is only 1 point - so I don't this is a very robust test.

Reply:
Done